easybuild/easyconfigs/g/gemini/gemini-0.30.2.eb

```diff
@@ -0,0 +1,34 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 		https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::		Robert Qiao <rob.qiao@flinders.edu.au>
+# Software License::	MIT
+#
+# Notes::
+##
+
+easyblock = 'Conda'
+
+name = 'gemini'
```

Comment:
Uppercase?

Reply:
`gemini` turns out to be a python package, I guess loser case is suitable? see what @Micket thinks?